blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -84,7 +91,7 @@ const ExtraLargePromo = ({ customFields }) => {
         <a
           href={content.website_url}
           className="xl-promo-headline"
-          title={content && content.headlines ? content.headlines.basic : ''}
+          title={headlineText}
```

Comment:
I'm assuming this doesn't have any side effects. looks like it's a small improvement https://github.com/WPMedia/fusion-news-theme-blocks/pull/507/files#diff-127d96d7bd4c23eafcd515680032e03291f78199b10f7bfb4f66c96f12e16524R54